src/saucelabs/request.js

```diff
@@ -39,4 +40,28 @@ export default class SaucelabsRequestAdapter {
 
         return body;
     }
+
+    async get (url) {
```

Comment:
`get` and `put` methods contains duplicated code. Move it to the `async _request(params)` function